tests/client.ts

```diff
@@ -0,0 +1,34 @@
+import { env } from "deno";
+import { test, assertEqual } from "https://deno.land/x/std@v0.2.6/testing/mod.ts";
+import { Client } from "../mod.ts";
+
+
+test(async function testDsnStyleParameters() {
```

Comment:
those tests should be replaced with unit tests for `ConnectionParams` - no need to do actual connection